crypto/hmac/hmac.c

```diff
@@ -18,16 +18,17 @@
 int HMAC_Init_ex(HMAC_CTX *ctx, const void *key, int len,
                  const EVP_MD *md, ENGINE *impl)
 {
-    int rv = 0;
-    int i, j, reset = 0;
+    int rv = 0, reset = 0;
+    int i, j;
```

Comment:
Kind of odd moving this up a line.

Reply:
It looked more odd as it was before. Now there are two zero-initialized local variables in line 21 and two uninitialized counter variables in line 22.

Reply:
I'm not suggesting it gets changed back, I agree it's better this way.  I just thought it odd to make a non-essential change.

Reply:
At one point during developing this change I removed the `reset` variable altogether. I later realised I still needed it so put it back again. Inadvertently it went to a slightly different place. But since everyone seems ok with it, and even to prefer it this way, I'll leave it.